charts/opensearch/CHANGELOG.md

```diff
@@ -13,13 +13,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 ### Security
 ---
-## [2.13.3]
+## [2.14.0]
 ### Added
 ### Changed
 ### Deprecated
 ### Removed
 ### Fixed
+- Updated OpenSearch appVersion to 2.9.0
+### Security
+---
+## [2.13.3]
+### Added
 - Correct variable name in keystore import
+### Changed
+### Deprecated
+### Removed
+### Fixed
 ### Security
```

Comment:
This is wrong, the original `fixed` of 2.13.3 is now `added`, while the new 2.14.0 `added` is now `fixed`.

Reply:
Thanks, updated in my latest commit.